From ff6e76347a265540354e565accad0792f56bb312 Mon Sep 17 00:00:00 2001
From: chandavath <rcrajagopal@habana.ai>
Date: Mon, 12 Feb 2024 10:55:43 +0200
Subject: [PATCH] [SW-137121] gaudi3: erom pcie: replaced fw sram base macro

FW_SRAM_BASE macro is defined in both gaudi2 and gaudi3 device
specific common.h file, so replaced newly added macro.

Change-Id: Ic923d4646c3c0e3d6d3edf9d5dce93c131b94c0a
Signed-off-by: chandavath <rcrajagopal@habana.ai>
---

diff --git a/app/ext_rom/erom_pcie.c b/app/ext_rom/erom_pcie.c
index c12c32a..4d9fdc3 100644
--- a/app/ext_rom/erom_pcie.c
+++ b/app/ext_rom/erom_pcie.c
@@ -32,11 +32,6 @@
 #define FLASH_VALID_MASK		0x1FFFFF
 #define SECUREBOOT_MAX_PPBOOT_IMG_SIZE	46080
 #define SPI_DMA_DST_FW_SRAM		BIT(1)
-#define HL_FWSRAM_OFFSET		0x3FD0000
-#define HL_GEN_MEM_BASE			0x24000000
-
-#define HL_FWSRAM_BASE			(HL_GEN_MEM_BASE + \
-					 HL_FWSRAM_OFFSET)
 #define PPBOOT_IMAGE_OFFSET		12
 #define PPBOOT_IMAGE_CODE_ALIGN		4
 #define PPBOOT_IMAGE_CODE_OFFSET	(PPBOOT_IMAGE_OFFSET + \
@@ -213,7 +208,7 @@
 			ret = erom_get_ppboot_data(ARCBOOT_SECONDARY);
 
 		if (!ret) {
-			ppboot = (void *)(HL_FWSRAM_BASE +
+			ppboot = (void *)(FW_SRAM_BASE +
 					  PPBOOT_IMAGE_CODE_OFFSET);
 			ppboot();
 		}
